src/Resources/public/vendor/admin-lte/yarn.lock

```diff
@@ -2708,7 +2708,7 @@ http-signature@~1.1.0:
     jsprim "^1.2.2"
     sshpk "^1.7.0"
 
-i@0.3.x:
```

Comment:
this should not be done, the same for the whole file.

Reply:
Fixed